src/test/java/com/wikia/webdriver/common/core/helpers/ArticlePurger.java

```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.common.core.helpers;
 
+import com.wikia.webdriver.common.logging.PageObjectLogging;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.logging.PageObjectLogging'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)